include/picolibrary/microchip/avr/megaavr/register.h

```diff
@@ -138,6 +138,54 @@ class Register {
     Type volatile m_register;
 };
 
+/**
+ * \brief A Microchip AVR megaAVR reserved register.
+ *
+ * \tparam The register's underlying integral type.
```

Comment:
Template parameter is not correctly documented. This should be `\tparam T The ...`. `::picolibrary::Microchip::AVR::megaAVR::Register` has the same problem.

Clarify that this underlying integral type is for a reserved register.